### PR TITLE
re-introduce envs files

### PR DESCRIPTION
**Pull request by a-thaler**
**Description**
With the migration of the prow pipelines to ADO we had to remove the env files in https://github.com/kyma-project/third-party-images/pull/425
Unfortunately, we missed the fact that the envs file is also used for creating the final docker tag. As there is no good solution, the envs file support will be added to image-builder ADO backend: https://github.com/kyma-project/test-infra/issues/10413

Changes proposed in this pull request:

- re-introduce env files
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: otel-collector/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-ARG GOLANG_VERSION=1.22
+ARG GOLANG_VERSION
```
Comment (Sawthis) [inline otel-collector/Dockerfile:1]: env file with GOLANG_VERSION is missing

**Review by Sawthis — CHANGES_REQUESTED** [review]

File: otel-collector/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
 FROM golang:${GOLANG_VERSION}-alpine as build
-ARG  OTEL_VERSION=0.97.0
+ARG  OTEL_VERSION
```
Comment (Sawthis) [inline otel-collector/Dockerfile:4]: env file with OTEL_VERSION is missing

**Review by Sawthis — CHANGES_REQUESTED** [review]

File: otel-collector/Dockerfile (hunk 1 above)
Comment (a-thaler) [reply otel-collector/Dockerfile:1]: I forgot to delete it in the previous PR, so it is still present in the repo

**Review by Sawthis — APPROVED** [review]

**Review by chrkl — APPROVED** [review]

**Comment by halamix2** [comment]
- Linking to https://github.com/kyma-project/test-infra/pull/10540
- and https://github.com/kyma-project/test-infra/pull/10548

**Review by halamix2 — APPROVED** [review]

**Review by lumi017 — APPROVED** [review]

**Pull request merged**